modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics using FlowMetrics API
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
+. Use the *Project:*  dropdown list to select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Here are a couple of examples of `FlowMetrics` configurations you can add:
++
+.Generate a metric that tracks ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+  namespace: netobserv                              <1>
+spec:
+  metricName: cluster_external_ingress_bytes_total  <2>
+  type: Counter                                     <3>
+  valueField: Bytes
+  direction: Ingress                                <4>
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType] <5>
+  filters:                                          <6>
+  - field: SrcSubnetLabel
+    matchType: Absence
+----
+<1> The same namespace, `netobserv`, is used across all example components in this documentation. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
```

Comment:
I find the 2nd and 3rd sentences contradicting each other. Or may be this can be rephrased better. The first sentence can be skipped.

Reply:
+1, we should simplify this.  thanks!

Reply:
Thanks, all :)